enforce_codeowners/keras_bot/pull_requests.py

```diff
@@ -78,14 +96,25 @@ def examine_single_pull_request(pull_request, map_path_owner):
         # let's not notify multiple people, otherwise it's going to turn into
         # a mess for big PRs with multiple files.
         return
+    cmd = "git clone --single-branch --branch master git@github.com:" + pull_request.user.login + "/" + repo_id + ".git"
+
+    os.system(cmd)
+    
+    absoluteOnly = import_tester.check_imports("./" + repo_id + "/keras")

```

Comment:
The current bot actually runs only keras-contrib. So I believe that this will fail as there is no `./keras` directory in keras-contrib.